Library/HUMSlider.m

```diff
@@ -96,6 +125,32 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder
 
 #pragma mark - Ticks
 
+- (void)addTick:(Tick*)tick willRefreshView:(bool)refreshView {
+    if ([self.ticks count] == 0) {
+        [self.ticks addObject:tick];
+    }
+    else {
+        unsigned long index = [self.ticks count];
+        for (Tick *tick_itr in [self.ticks reverseObjectEnumerator]) {
```

Comment:
styling nit: Obj-C code generally uses `camelCase` instead of `snake_case` for variable names